packages/documentation-site/patternfly-docs/content/demos/Compass/CompassIntegrationsDemo.tsx

```diff
@@ -0,0 +1,750 @@
+      statusText: "Connected",
+      type: "Cache",
+      url: "redis.example.com",
+    },
+    {
+      id: 10,
+      name: "PostgreSQL Database",
+      description:
+        "PostgreSQL is a powerful, open source object-relational database system.",
+      status: "success",
+      statusText: "Connected",
+      type: "Database",
+      url: "postgres.example.com",
+    },
+    {
+      id: 11,
+      name: "Apache Kafka",
+      description:
+        "Apache Kafka is a distributed event streaming platform for building real-time data pipelines.",
+      status: "warning",
+      statusText: "Limited",
+      type: "Message Queue",
+      url: "kafka.example.com",
+    },
+    {
+      id: 12,
+      name: "Terraform",
+      description:
+        "Terraform is an infrastructure as code tool for building, changing, and versioning infrastructure.",
+      status: "success",
+      statusText: "Connected",
+      type: "Infrastructure",
+      url: "terraform.example.com",
+    },
+    {
+      id: 13,
+      name: "Splunk",
+      description:
+        "Splunk is a platform for searching, monitoring, and analyzing machine-generated big data.",
+      status: "danger",
+      statusText: "Disconnected",
+      type: "Logging",
+      url: "splunk.example.com",
+    },
+  ];
+
+  const handleSendMessage = () => {
+    setIsThinking(true);
+    setTimeout(() => {
+      setIsThinking(false);
+    }, 10000); // 10 seconds
+  };
+
+  const navContent = (
+    <>
+      <CompassPanel isPill hasNoPadding>
+        <Tabs
+          activeKey={1}
+          isNav
+          onSelect={() => {}}
+          component={TabsComponent.nav}
+          aria-label="Compass navigation tabs"
+          inset={{ default: 'insetXl' }}
+        >
+          <Tab eventKey={0} title={<TabTitleText>Tab 1</TabTitleText>} />
+          <Tab eventKey={1} title={<TabTitleText>Integrations</TabTitleText>} />
+          <Tab eventKey={2} title={<TabTitleText>Tab 3</TabTitleText>} />
+          <Tab eventKey={3} title={<TabTitleText>Disabled Tab 4</TabTitleText>} isDisabled />
+        </Tabs>
+      </CompassPanel>
+    </>
+  );
+
+  const startSidebarContent = (
+    <CompassPanel isPill>
+      <ActionList isIconList isVertical>
+        <ActionListGroup>
+          <ActionListItem>
+            <Tooltip content="Play">
+              <Button variant="plain" icon={<PlayIcon />} aria-label="Play" isCircle />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Add">
+              <Button variant="plain" icon={<OutlinedPlusSquare />} aria-label="Add" isCircle />
+            </Tooltip>
+          </ActionListItem>
+        </ActionListGroup>
+        <ActionListItem>
+          <Tooltip content="Chat with AI">
+            <Button variant="plain" icon={<Icon style={{translate: ".05em .1em; scale: 1.1"}}><RHAiExperienceIcon /></Icon>} aria-label="Chat" isCircle className="pf-v6-m-ai-indicator" />
```

Comment:
```suggestion
            <Button variant="plain" icon={<Icon style={{translate: ".05em .1em; scale: 1.1"}}><RHAiExperienceIcon /></Icon>} aria-label="Chat with AI" isCircle className="pf-v6-m-ai-indicator" />
```

Just to have the name match the tooltip